### ISSUE TITLE
Need an example of metric calculation on custom video pair

### ISSUE THREAD

**Issue by Kepnu4** [hqqxyy/VFIPS#2]
It would be great to have a sample on how to calculate metric on custom video pair. I tried to adopt it from test sample but got negative values. What is the expected range for metric?

**Comment by hqqxyy** [comment]
Hi Kepnu4,

At present, our method does not have a defined range for the metric. In our experiments, we compared the rank of our scores with the rank of human scores, resulting in the possibility of negative scores.

I plan to fine-tune a version that constrains the score range to [0, 1] within the next few days. I will keep you updated on my progress and notify you once this modification is complete.

Thank you for your understanding.

Best regards,
Qiqi

**Comment by basilevh** [comment]
Hi,
I am also getting negative values even for reasonably good video pairs, which strikes me as weird because the results in the [LDM-VFI paper](https://arxiv.org/pdf/2303.09508.pdf) are all positive and also close to 100. Do you happen to have any update about the fine-tuned version you mentioned?
Thanks in advance!

**Comment by hqqxyy** [comment]
Hi,

Sorry. I forgot this issue. @basilevh No, we have not updated our metrics yet. For the LDM-VFI, you might need to contact the authors of LDM-VFI for the details. To evaluate their method, they need to repeatedly interpolate their results and other operations. 

Thank you!

**Comment by basilevh** [comment]
Hi @hqqxyy, I am consistently observing that the better the quality of my predicted video (including that of baselines), the closer the result of VFIPS is to -100. It is almost always negative (between 0 and -100). Would you mind confirming that this is expected? Thanks again.

**Comment by hqqxyy** [comment]
@basilevh  Yes. Our metric should get lower scores when the predicted video has a higher perceptual quality.
